cpp/IceBox/greeter/Client.cpp

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) ZeroC, Inc.
+
+#include "Greeter.h"
+
+#include <cstdlib>
+#include <iostream>
+
+using namespace std;
+
+int
```

Comment:
A simplified version of the Ice/greeter client. You can also use the Greeter client if you like.